### PR TITLE
[Doc Improvement][Teams AI library][3706562]

### PR DESCRIPTION
**Pull request by v-bvishnu**
Python codes are updated.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8692a671963721c200e93f82529e1a89064a7f88/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 86 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c3c57ecf-a5e9-49eb-b7eb-53bc8e2f63ba) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8692a671963721c200e93f82529e1a89064a7f88/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b8954cc5-548e-4d45-9a6f-953cfc9cd3fb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8692a671963721c200e93f82529e1a89064a7f88/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ba9990cb-e84a-4b9a-a5be-320f85c6ecce) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/82f5c427383a324b01959664c7a426ca3b255a8c/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 86 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/36c0927a-a837-448a-8a9a-056dd7388116) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/82f5c427383a324b01959664c7a426ca3b255a8c/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fa3fe962-c292-4770-8afa-965476a51969) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/82f5c427383a324b01959664c7a426ca3b255a8c/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2e5709d4-e468-4540-9ce7-72e04e51ba35) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5d12678ec5551723fe1499edea3b7855120a630f/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 90 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6f09606e-e113-45e8-9c85-e83fe4e4a21b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5d12678ec5551723fe1499edea3b7855120a630f/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/96f99f75-7d56-4ccb-af68-b303d9be72bd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5d12678ec5551723fe1499edea3b7855120a630f/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eb49edbf-6e90-4c0d-aad5-4d0d2cdf8877) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b11e3ec51061da018e0463967a6d0b193613a0b9/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 90 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2f6db996-ec24-4e0c-9794-1196f7ad22ee) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b11e3ec51061da018e0463967a6d0b193613a0b9/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/14fb76e5-1f44-4266-ac1b-1c0901d7b72e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b11e3ec51061da018e0463967a6d0b193613a0b9/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0ebf9c7e-5e3e-4a81-b7dc-007fd6f5252d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md [hunk 1]
```diff
@@ -120,23 +120,23 @@ Teams AI Library offers an  integrated fact-checking system to tackle bot halluc
...
+|Prompts     | A new object-based prompt system enables better token management and reduces the likelihood of overflowing the model's context window.        | ✔️ |✔️|✔️|
+| Augmentation     | Augmentations simplify prompt engineering tasks by letting the developer add named augmentations to their prompt. Only  `functions`, `sequence`, and `monologue` style augmentations are supported.        | ✔️ |✔️|✔️|
+|Data Sources     |  A new DataSource plugin makes it easy to add RAG to any prompt. You can register a named data source with the planner and then specify the name[s] of the data sources they wish to augment the prompt.      | ❌ |✔️|✔️|
 
 ## Code samples
 
-|Sample name | Description |.NET | Node.js |
-|----------------|-----------------|--------------|--------------|
-|Echo bot| This sample shows how to incorporate a basic conversational flow into a Microsoft Teams application using Bot Framework and the Teams AI library. | [View](https://github.com/microsoft/teams-ai/tree/main/dotnet/samples/01.messaging.echoBot) | [View](https://github.com/microsoft/teams-ai/tree/main/js/samples/01.messaging.a.echoBot) |
+|Sample name | Description |.NET | Node.js | Python|
```
Comment (lilyydu) [inline msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md:137]: In addition to Echo bot, please update so we have checkmarks next to these 2 bots for Python:

- Message extensions: GPT-ME
- Light bot

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/61e343402a58b80bf92fdce461d997f9744ada45/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 90 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/19b70404-f00f-41d7-8262-6d3588a2387a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/61e343402a58b80bf92fdce461d997f9744ada45/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/428d7e07-46e5-4190-9736-ca271af5f7bc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/61e343402a58b80bf92fdce461d997f9744ada45/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/59c3c45b-d787-4910-88c1-1c7165ec7d9e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md (hunk 1 above)
Comment (v-bvishnu) [reply msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md:137]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0cd6b08bf192343cab67da237491c350f2cd081f/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 90 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5ffc2da4-eefd-42f9-828d-22608eb5294e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0cd6b08bf192343cab67da237491c350f2cd081f/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0024b0ae-378b-401b-ae2e-8a969ba99626) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0cd6b08bf192343cab67da237491c350f2cd081f/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/05796110-e937-47ba-b1a3-ee6c1277e899) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0cd6b08bf192343cab67da237491c350f2cd081f/msteams-platform%2Fwhats-new.md) | 91 | 73 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e2170a84-c125-4634-95d9-d192f3eba126) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f87754151027d4843f71c4fd06402df38b1dc88a/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 90 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/51d5ca68-54df-4db5-b634-f485569c89c6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f87754151027d4843f71c4fd06402df38b1dc88a/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/789aec0e-ec0d-49de-b4f6-b46e858faf90) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f87754151027d4843f71c4fd06402df38b1dc88a/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5af1d23e-8edc-4ef0-9489-65b2b4ecd354) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f87754151027d4843f71c4fd06402df38b1dc88a/msteams-platform%2Fwhats-new.md) | 91 | 73 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6d01e661-8564-4d9f-b177-30057629cfe1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by lilyydu — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fddbaea582f93fc2fd493897098384fd43452b44/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 90 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/06e171da-5e2f-4065-abb8-69bc0328e695) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fddbaea582f93fc2fd493897098384fd43452b44/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aa304911-5857-49a0-9f52-695e353187f2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fddbaea582f93fc2fd493897098384fd43452b44/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/70eefa7b-f34d-4639-bbb2-339e16c0d204) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fddbaea582f93fc2fd493897098384fd43452b44/msteams-platform%2Fwhats-new.md) | 90 | 76 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6ff63760-e869-4a80-a9b2-36d2746be850) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c7cc71a666086efe234ab9a41789399ac1f62f72/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 90 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9e4a00a4-d7e7-46c5-8452-93d934db5940) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c7cc71a666086efe234ab9a41789399ac1f62f72/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c98395c2-8ab4-4cd2-83b3-03dd4429340f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c7cc71a666086efe234ab9a41789399ac1f62f72/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/97542009-5514-42a2-b0eb-97d318d42ea9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/c7cc71a666086efe234ab9a41789399ac1f62f72/msteams-platform%2Fmstdd-landing.yml) | 85 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/12d72d55-0156-4634-a0eb-e49cc2a08dc2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c7cc71a666086efe234ab9a41789399ac1f62f72/msteams-platform%2Fwhats-new.md) | 90 | 76 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/37d43482-ac80-40af-bc1a-4ec820c1a121) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by govil1811 — DISMISSED** [review]

**Review by surbhigupta12 — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-core-capabilities.md](https://github.com/MicrosoftDocs/msteams-docs/blob/da6e726419fd3c5c3b4accaba84c7707956d276e/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-core-capabilities.md) | 90 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0e2f8a42-cbdd-4c94-b9c4-825e3e2fe8b9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md](https://github.com/MicrosoftDocs/msteams-docs/blob/da6e726419fd3c5c3b4accaba84c7707956d276e/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fhow-conversation-ai-get-started.md) | 90 | 69 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b2b0b2f4-2806-4665-9ad7-5494db242e40) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/Teams conversational AI/teams-conversation-ai-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/da6e726419fd3c5c3b4accaba84c7707956d276e/msteams-platform%2Fbots%2Fhow-to%2FTeams%20conversational%20AI%2Fteams-conversation-ai-overview.md) | 92 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fc655bc6-3e2a-46f3-82b8-166e991708b9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/da6e726419fd3c5c3b4accaba84c7707956d276e/msteams-platform%2Fmstdd-landing.yml) | 81 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e4dfb653-c952-41a6-8a4f-4f0d588ef270) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/da6e726419fd3c5c3b4accaba84c7707956d276e/msteams-platform%2Fwhats-new.md) | 90 | 76 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7d4acb33-4d98-4232-b728-9ef9c58aeaec) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**